src/x86/uarch.c

```diff
@@ -149,16 +153,30 @@ struct uarch* get_uarch_from_cpuid_intel(uint32_t ef, uint32_t f, uint32_t em, u
   // ------------------------------------------------------------------------------- //
   //                EF  F  EM   M   S                                                //
   UARCH_START
+  CHECK_UARCH(arch, 0,  4,  0,  0, NA, "i80486DX",          UARCH_I486,            UNK) // sandpile.org
```

Comment:
Can you provide the exact sandpile.org URL that has these CPU values?

Reply:
https://sandpile.org/x86/cpuid.htm#level_0000_0001h

Under the 'model' section of the giant table

Reply:
Right, but can you add it as a comment? It's important to keep these things documented.

Reply:
Sure. Do you just want each of those CPUs to have a comment with the same URL?

Reply:
Having each one with a comment pointing to the same URL (what you have right now) is fine.